src/main/java/Duke.java

```diff
@@ -1,10 +1,225 @@
+import java.util.HashMap;
+import java.util.Scanner;
+
 public class Duke {
+    private static boolean isListening;
+    private static Task[] tasks = new Task[100];
+    private static int numTasks;
+
+
+    public static void setIsListening(boolean keepListening) {
+        Duke.isListening = keepListening;
+    }
+
+    public static void PrintBorder() {
+        System.out.println("────────────────────────────────────────────────────────────");
+    }
+
+    public static void Exit() {
```

Comment:
camelCase standard : it should be "exit"
```suggestion
    public static void exit() {
```